src/main.ts

```diff
@@ -7,8 +7,9 @@ Vue.use(Vuetify);
 
 let v = new Vue({
     el: "#app",
+    vuetify: new Vuetify(),
```

Comment:
This should have been working from line 6 `Vue.use(Vuetify)` was there a conflict?

Reply:
There was no conflict but when I tried using v-select from Vuetify but it didn't work so I did some research and figured I could add Vuetify instance and it worked after I did. The error was 'Cannot read property t of undefined'.

Reply:
Cool, then I think we can remove the use declaration on line 6.

Reply:
I just tried removing Vue.use(Vuetify) and it's not working without it. It seems it needs Vue.use(Vuetify) somehow.